SportsHub/SportsHub.Application/Services/FilterService.cs

```diff
@@ -0,0 +1,20 @@
+﻿using SportsHub.Domain.Models;
+using SportsHub.Domain.UOW;
+
+namespace SportsHub.AppService.Services
+{
+    public class FilterService : IFilterService
```

Comment:
No need to add a new service to get items by a filter. This can be done in Article service.